### ISSUE TITLE
UnboundLocalError

### ISSUE THREAD

**Issue by carter2422** [CGCookie/script-bakery#99]
Every now and then I'm getting this error:
http://cl.ly/Qt6e

```
Traceback (most recent call last):
  File "/Users/jonathanwilliamson/Dropbox/resources/repositories/cgCookie/script-bakery/scripts/addons/contour_tools/__init__.py", line 845, in modal
    h_target = c_cut.active_element(context,event.mouse_region_x,event.mouse_region_y)
  File "/Users/jonathanwilliamson/Dropbox/resources/repositories/cgCookie/script-bakery/scripts/addons/contour_tools/../contour_tools/contour_classes.py", line 965, in active_element
    elif active_self:
UnboundLocalError: local variable 'active_self' referenced before assignment
```

I can't seem to narrow down why it happens. Sometimes it happens on a model and then next time it works fine.

**Comment by patmo141** [comment]
Hmm that's an odd one. That's old old selection code. Been in there since way back at initial user interface code.

**Comment by wchargin** [comment]
FYI commit d2a5367, with message "should fix #99 but will leave open," automatically closed the issue because it saw the text "fix #99" ... you may wish to reopen.
